framework/src/modules/chain/blocks/blocks.js

```diff
@@ -293,6 +294,7 @@ class Blocks extends EventEmitter {
 				this.storage,
 				this.interfaceAdapters,
 				this.genesisBlock,
+				blocksLogic,
```

Comment:
Agree that its the easiest way to solve. But I don't like that direction. We have to resolve and not pass any other module to utils. To me when util functions require instance of other modules then these are actually not standalone utils functions. We should rather move `loadLastBlock` to `blockLogic` where it relates more accurately.

Reply:
The loadLastBlock function is not the issue, it is that `dbRead` and `storageRead` require `blocksLogic`.